index.html

```diff
@@ -5,11 +5,45 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" type="text/css" href="style.css">
-        <title>Hello Microverse</title>
+        <title>Portfolio Setup Mobile First</title>
     </head>
     <body>
-        <header>
-        <h1 class="header">Hello Microverse!</h1>
+        <!--Header section-->
+        <header class="header">
+            <a id="logo" href="#">Allan K. </a>
+            <nav>
+                <a href="#"><img class="menuimg" src="assets/header/menu.png" alt="menu"/></a>
+            </nav>
         </header>
+        <section class="content">
+            <main class="welcome">
+                <article>
+                    <h1 class="main-title">Hey There. I'm Allan.</h1>
+                    <h2 class="sub-title">I'm a Software Developer</h2>
+                    <p class="info">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
+                    <img class="bg-img" src="assets/background/bg_img.png" alt="bg-img"/>
+                </article>
+                <footer>
+                    <ul class="contact-links">
+                        <li>
+                            <a href="https://github.com/allenkays"><img  id="profile" src="assets/contact_profiles/github.png"/></a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/allan-sang/"><img  id="profile" src="assets/contact_profiles/linkedin.png"/></a>
+                        </li>
+                        <li>
+                            <a href="https://discord.com/allenkays"><img  id="profile" src="assets/contact_profiles/angelist.png"/></a>
+                        </li>
+                        <li>
+                            <a href="https://twitter.com/allenkays"><img  id="profile" src="assets/contact_profiles/twitter.png"/></a>
+                        </li>
+                        <li>
+                            <a href="https://medium.com/@allainkibet"><img  id="profile" src="assets/contact_profiles/medium.png"/></a>
+                        </li>
+                    </ul>
+                </footer>
```

Comment:
- [ ] [OPTIONAL] I'd suggest you remove the footer tag from the landing page here as it's semantically incorrect and should only be reserved for the main footer of the page which you've yet gotten to implement. ✔

Reply:
Thanks. Let me do that